### PR TITLE
Switched to Lykke.Numerics.Money

### PR DESCRIPTION
**Pull request by PaterSantyago**

### CODE REVIEW

File: src/Lykke.Bil2.Contract/TransactionsExecutor/Requests/AssetFeeOptions.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿using JetBrains.Annotations;
-using Lykke.Bil2.Contract.Common;
 using Lykke.Bil2.Contract.Common.Exceptions;
+using Lykke.Bil2.Contract.Common.JsonConverters;
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Bil2.Contract/TransactionsExecutor/Requests/AssetFeeOptions.cs]: Remove this entire file, please. I missed it during contract update.

File: src/Lykke.Bil2.Contract/TransactionsExecutor/Requests/TransactionCoinsValidator.cs [hunk 2]
```diff
@@ -38,7 +38,7 @@ public static void Validate(IReadOnlyCollection<CoinToSpend> coinsToSpend, IRead
                 .Select(g => new
                 {
                     AssetId = g.Key,
-                    Sum = g.Sum(x => x.Value)
+                    Sum = g.Select(x => x.Value).Aggregate((x, y) => x + y)
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Bil2.Contract/TransactionsExecutor/Requests/TransactionCoinsValidator.cs]: Would be nice to have overloads of Sum() (and another Linq methods) which can work with Money/UMoney directly.

**Review by KonstantinRyazantsev — COMMENTED** [review]
Let's register Money and UMoney Json converters in the JsonConvert.DefaultSettings and Web API GlobalConfiguration.Configuration.Formatters.JsonFormatter.SerializerSettings and avoid specifying it everywhere aruound the code.

**Comment by PaterSantyago** [comment]
Added TypeConverters to Money and UMoney in Lykke.Numerics.Money, so JsonConverters for these types are not necessary anymore.

**Review by KonstantinRyazantsev — DISMISSED** [review]

**Review by OttoVT — APPROVED** [review]

**Review by cassade — APPROVED** [review]

**Pull request merged**